client/main.go

```diff
@@ -59,6 +59,7 @@ func main() {
 			return fmt.Errorf("error initializing ark sdk client: %v", err)
 		}
 		arkSdkClient = sdk
+
```

Comment:
Are we sure in using a global variable here? Wouldnt be better to have it inside the context?

Reply:
The SDK exposes `IsLocked` https://github.com/sekulicd/ark/blob/52b21c5e687111250b365e89438503e2327ce107/pkg/client-sdk/wallet/wallet.go#L21